templates/shortcodes/notice_box.md

```diff
@@ -1,3 +1,3 @@
 <div style="text-align: center; padding: .4em 1em; margin-bottom: 1rem; background-color: #ffcb006b;">
```

Comment:
I was wondering if we wanted to hardcode the style in the component.

- Pros: we have a single place to look for everything notice_box, in a component-like approach
- Cons: not really the usual way we style things, but I don't think there are many "performance" gains to make by moving it to a css file

So I guess let's keep it here?

Reply:
Fair criticism; I think I may have accidentally followed the pattern from templates/shortcodes/figure.html by copying it.

Fixed in 606d0067